docs/build-n-test.md

```diff
@@ -13,12 +13,14 @@
 ## Build Instructions
 * dcurl allows various combinations of build configurations to fit final use scenarios.
 * You can execute `make config` and then edit file `build/local.mk` for custom build options.
-    - ``BUILD_AVX``: build Intel AVX-accelerated Curl backend.
-    - ``BUILD_GPU``: build OpenCL-based GPU accelerations.
+    - ``BUILD_AVX``: build the Intel AVX-accelerated Curl backend.
+    - ``BUILD_SSE``: build the Intel SSE-accelerated Curl backend.
+    - ``BUILD_GENERIC``: build the generic CPU Curl backend without acceleration.
```

Comment:
Move the description about `BUILD_GENERIC` out of custom build options. Instead, it should be mentioned as the fallback.

Reply:
I will create a paragraph of `Fallback Build Instructions` in the document.